aclk/tests/fake_charts.d.plugin

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+sleep 45    # Wait until popcorning finishes
+
+echo "CHART aclk_test.newcol '' 'Generate new collector/chart event' 'units' aclk_test aclk_test lines 900001 1"
+sleep 5
+echo "DIMENSION aclk1 '' percentage-of-absolute 1 1"
+sleep 5
+echo "BEGIN aclk_test.newcol 1000000"
+echo "SET aclk1 = 3"
+echo "END"
+sleep 5
+echo "DIMENSION aclk2 '' percentage-of-absolute 1 1"
+sleep 5
+echo "BEGIN aclk_test.newcol 1000000"
+echo "SET aclk1 = 3"
+echo "SET aclk2 = 3"
+echo "END"
+sleep 5
+echo "CHART aclk_test2.newcol '' 'Generate new collector/chart event' 'units' aclk_test aclk_test lines 900001 1"
+echo "DIMENSION aclk1 '' percentage-of-absolute 1 1"
+
+sleep 5
+exit 0   # Signal that we are done
```

Comment:
the plugin will be restarted by netdata if exit code is 0 if i remember it right, not sure it is important for the test case, just saying

Reply:
Might be the case - I remember seeing a message in the logs that said the collector had terminated and would not be restarted. If this is not the case during testing then flip the return code and see if we can provoke the collector deleted case.